Jenkinsfile

```diff
@@ -20,7 +20,7 @@ properties([
 ])
 currentBuild.rawBuild.getParent().setQuietPeriod(0)
 
-library 'jenkins-pipeline-library@main'
+library 'jenkins-pipeline-library@CRE-938'
```

Comment:
We should merge jenkins-pipeline-library to main before merging this and swap it back to main.

Reply:
According to @ETSolace the jenkins-pipeline-library changes are merge to main so this shoudl be udpated on the branch first. I also got to handle conflicts from dev. My next will fix all of this.